docs/book/generator/examples.md

```diff
@@ -370,3 +370,46 @@ $generator->addMethod(
 );
 $code = $generator->generate();
 ```
+
+
+## Generating Trait 
+
+`Laminas\Code\Generator\FileGenerator` can be used to generate the contents of a *PHP* file. You can
+include classes as well as arbitrary content body.
+
+```php
+use Laminas\Code\Generator\FileGenerator;
+use Laminas\Code\Generator\TraitGenerator;
+
+$traitclass = new TraitGenerator();
+$traitclass->addMethod('MethodOne');
+$traitclass->addConstant('VERSION','1.0.0');
+
+$traitFile = new Laminas\Code\Generator\FileGenerator();
+$traitFile->setClass($traitclass);
+
+```
```

Comment:
The code example can be shortened to the essentials.
```suggestion
```php
$trait = new Laminas\Code\Generator\TraitGenerator();
$trait->setName('TranslatorAwareTrait');
$trait->addProperty(
    'translator',
    null,
    Laminas\Code\Generator\PropertyGenerator::FLAG_PRIVATE
);
$trait->addMethod('getTranslator');

echo $trait->generate();
```
```

Reply:
I think the `use` statements could still be handy - though I admit I don't know how the rest of the documentation is structured and if there's a standard or not.